src/components/Dashboard/FilingHistoryList.vue

```diff
@@ -379,346 +64,97 @@
+    ...Filings,
     AddCommentDialog,
     DownloadErrorDialog,
     FileCorrectionDialog,
     LoadCorrectionDialog
   },
-  mixins: [
-    AllowableActionsMixin,
-    DateMixin,
-    FilingMixin
-  ]
+  mixins: [FilingMixin]
 })
 export default class FilingHistoryList extends Vue {
   @Prop({ default: null }) readonly highlightId!: number
 
+  @Getter getCurrentFiling!: ApiFilingIF
   @Getter getEditUrl!: string
   @Getter getFilings!: Array<ApiFilingIF>
-  @Getter hasCourtOrders!: boolean
+  @Getter getIdentifier!: string
+  @Getter getPanel!: number
   @Getter isBenBcCccUlc!: boolean
   @Getter isFirm!: boolean
   @Getter isRoleStaff!: boolean
+  @Getter hasCourtOrders!: boolean
+  @Getter showAddCommentDialog!: boolean
+  @Getter showDownloadErrorDialog!: boolean
+  @Getter showLoadCorrectionDialog!: boolean
+  @Getter showFileCorrectionDialog!: boolean
 
-  @Action setIsCoaPending!: ActionBindingIF
-  @Action setCoaEffectiveDate!: ActionBindingIF
-  @Action setHasBlockerFiling!: ActionBindingIF
-
-  // local properties
-  protected addCommentDialog = false
-  protected downloadErrorDialog = false
-  protected fileCorrectionDialog = false
-  protected loadCorrectionDialog = false
-  protected panel = null as number // currently expanded panel
-  protected historyItems: Array<HistoryItemIF> = []
-  protected loadingOne = false
-  protected loadingAll = false
-  protected currentFiling = null as HistoryItemIF
-  protected loadingOneIndex = -1
-  protected isBusy = false
+  @Action hideCommentDialog!: ActionBindingIF
+  @Action setFetchingDataSpinner!: ActionBindingIF
+  @Action toggleFilingHistoryItem!: ActionBindingIF
 
-  // for template
-  readonly AllowableActions = AllowableActions
-  readonly EnumUtilities = EnumUtilities
+  @Mutation mutateDownloadErrorDialog!: (x: boolean) => void
+  @Mutation mutateFileCorrectionDialog!: (x: boolean) => void
+  @Mutation mutateLoadCorrectionDialog!: (x: boolean) => void
 
-  /** The Temporary Registration Number string. */
-  get tempRegNumber (): string {
-    return sessionStorage.getItem('TEMP_REG_NUMBER')
+  /** Whether this entity is a business (and not a temporary registration). */
+  get isBusiness (): boolean {
+    return !!sessionStorage.getItem('BUSINESS_ID')
   }
 
-  /** The Business ID string. */
-  get businessId (): string {
-    return sessionStorage.getItem('BUSINESS_ID')
+  /** Whether this entity is a temporary registration (and not a business). */
+  get isTemporaryRegistration (): boolean {
+    return !!sessionStorage.getItem('TEMP_REG_NUMBER')
   }
 
   /** Whether to show the history panel. */
   get showHistoryPanel () {
-    return (this.historyItems.length > 0)
-  }
-
-  /** Returns whether the action button is visible or not. */
-  protected displayAction (filing): string {
-    return filing.availableOnPaperOnly || filing.isTypeStaff || filing.documentsLink
-  }
-
-  /** Loads list of filings from the API into History Items array. */
-  private loadData (): void {
```

Comment:
Both loadData() and loadFiling() are obsolete. The template now iterates over getFilings (from store).